main.tf

```diff
@@ -253,7 +253,7 @@ resource "aws_appautoscaling_target" "ecs" {
 
   max_capacity       = lookup(var.appautoscaling_settings, "max_capacity", var.desired_count)
   min_capacity       = lookup(var.appautoscaling_settings, "min_capacity", var.desired_count)
-  resource_id        = "service/${var.cluster_id}/${aws_ecs_service.this.name}"
+  resource_id        = "service/${var.cluster_name}/${aws_ecs_service.this.name}"
```

Comment:
@ziazek naming is a bit ambiguous here, but we're always passing the cluster name of an ECS cluster as `var.cluster_id` to this module which has been working for all resources incl. app autoscaling so far (we should update our example and the description of the variable though).

Could you please test using the name instead of the ARN?

Reply:
@moritzzimmer ah, it works that way. Please ignore this pull request.